### ISSUE TITLE
about training time

### ISSUE THREAD

**Issue by chenyuyuyu** [Delay-Xili/F-Clip#7]
Hello, I want to know how long it took you to train FClip-HR? It took me more than seven days. Is there any way to reduce the training time?

**Comment by Delay-Xili** [comment]
Hi,
If you are not pursuing the state of the art performance, just set the training epoch to 35 and make the learning rate decay at 25 and 30 respectively, that will be enough for you to obtain acceptabel results.

**Comment by Sun15194** [comment]
Hello, I have modified the base.yaml and fclip_HR.yaml file：
lr: 4.0e-3
max_epoch: 35
lr_decay_epoch: [25,30]
weight_decay: 8.0e-4

However, the results were not so good:
f1 measure is: 0.33950091807137656
AP-H is: 0.19708869533106066

I noticed in the paper that you used a single NVIDIA GTX 2080Ti GPU to train for 300 epochs. How long did this process take?

**Comment by lucy3589** [comment]
> Hi, If you are not pursuing the state of the art performance, just set the training epoch to 35 and make the learning rate decay at 25 and 30 respectively, that will be enough for you to obtain acceptabel results.

if train HG1_D2 model.  How many epochs need to be trained?
